Assets/SEE/UI/SettingsMenu.cs

```diff
@@ -58,6 +156,20 @@ protected override void UpdateDesktop()
             }
         }
 
+        /// <summary>
+        /// The keyBinding which gets updated
+        /// </summary>
+        string buttonToRebind = null;
+
+        /// <summary>
+        /// Sets the <see cref="buttonToRebind"/>.
+        /// </summary>
```

Comment:
Document the parameter `buttonName`. Is `buttonName` the right term to be used here?